synapse/http/matrixfederationclient.py

```diff
@@ -649,12 +651,12 @@ async def _send_request(
 
                     if retries_left and not timeout:
                         if long_retries:
-                            delay = 4 ** (MAX_LONG_RETRIES + 1 - retries_left)
-                            delay = min(delay, 60)
+                            delay = 4 ** (self.max_long_retries + 1 - retries_left)
+                            delay = min(delay, self.max_long_retry_delay)
                             delay *= random.uniform(0.8, 1.4)
                         else:
                             delay = 0.5 * 2 ** (MAX_SHORT_RETRIES - retries_left)
-                            delay = min(delay, 2)
+                            delay = min(delay, self.min_retry_delay)
```

Comment:
I think this is better named `max_short_retry_delay`? It's the same as `max_long_retries` except it applies to things using the short retry scheme